### ISSUE TITLE
working in watch, but not when just running grunt

### ISSUE THREAD

**Issue by ronaldlokers** [khalillechelt/grunt-compass#1]
The code does work perfectly when you set up compass the right way and configurate grunt to run the task with the watch option. But it does not work when you configure it as a general task and then run it with "grunt" or "grunt compass"

It does mention "`compass compile` was initiated." in the console, but the only thing compass does is creating the folders, it doesn't create the css files.

This is the config file I used for grunt:

``` javascript
/*global module:false*/
module.exports = function(grunt) {

  // Project configuration.
  grunt.initConfig({
    compass: {
      files: ['assets/sass/**/*.scss'],
      tasks: 'compass'
    },
    watch: {
      files: '<config:compass.files>',
      tasks: 'compass'
    }
  });

  // Default task.
  grunt.registerTask('default', 'compass');

  grunt.loadTasks('grunt/grunt-compass/tasks');

};
```

**Comment by khalillechelt** [comment]
For now you'll have to configure compass yourself and use it as advertised ;)

You are more than welcome to fork and improve of course :) 

## 

Sent from my iPhone

Am Samstag, 2. Juni 2012 um 15:59 schrieb Ronald Lokers:

**Comment by ronaldlokers** [comment]
I did configure compass myself and it does work with "grunt watch", but I can't get it working with just running "grunt".

I am not sure what the difference is since the task only runs "compass compile", but with running "grunt" it will only create folders in the "css" folder and not the files that are in the "sass" folder.

To test some things I runned Compass without a config file and set everything in the call to the program. So I ended up with something like

```
compass compile "/path/to/project" --sass-dir="sass" --css-dir="css"
```

This does work with "grunt watch" and when directly called from the terminal, but this doesn't work when using it with "grunt".

I don't know what really is going wrong, but this looks like a bug in grunt itself.
